### PR TITLE
Publish monaco in arc

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
- Added needed dependencies
- Renamed the Monaco component
- Added the more default props
- Switched resize observable dependency to one without type errors :+1:

### CODE REVIEW

File: src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx [hunk 1]
```diff
@@ -27,19 +27,21 @@ import {
...
   !code.trimLeft().startsWith(':') &&
   !code.trimLeft().toUpperCase().startsWith('EXPLAIN') &&
   !code.trimLeft().toUpperCase().startsWith('PROFILE')
 
-export type MonacoHandles = Monaco
+export type CypherEditorHandles = CypherEditor
 
+/*
+// Using this gives "invalid hook call" for some reason
+import styled from 'styled-components'
```
Comment (OskarDamkjaer) [inline src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx:44]: Forgot to fix this.. On it!

File: src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx [hunk 2]
```diff
@@ -57,44 +55,68 @@ const MonacoStyleWrapper = styled.div`
...
 const EDITOR_UPDATE_DEBOUNCE_TIME = 300
-type MonacoDefaultProps = {
-  value: string
-  onDisplayHelpKeys: () => void
-  onChange: (value: string) => void
-  onExecute: (value: string) => void
-}
-
-export type MonacoProps = MonacoDefaultProps & {
+type CypherEditorDefaultProps = {
```
Comment (eijawerner) [inline src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx:58]: There is a Required util in typescript, which converts optionals to required, so suggest to ``type CypherEditorDefaultProps = Required<CypherEditorProps>`` to not have it defined twice.

File: package.json [hunk 3]
```diff
@@ -173,9 +172,10 @@
   },
   "dependencies": {
     "@apollo/client": "^3.2.0",
-    "@neo4j/browser-lambda-parser": "1.0.5",
+    "@juggle/resize-observer": "^3.3.1",
```
Comment (eijawerner) [inline package.json:175]: can we remove this all together since all browsers that we support now don't need this?

**Review by eijawerner — COMMENTED** [review]
Nice! ⭐ See comments.

**Review by eijawerner — APPROVED** [review]
💯

File: package.json (hunk 3 above)
Comment (OskarDamkjaer) [reply package.json:175]: I removed this but then noticed our typescript version didn't have types for the resize-observer yet, started upgrading it which meant I had to fix new type errors, install a new version of ts-jest and so on. Turned out to be too much of a rabbit hole for now, so I'd prefer to keep the polyfill in for now :+1:

**Pull request merged**